<?xml version="1.0"?>

<launch>

  <node name="dawn_ik_rviz_control" pkg="dawn_ik" type="dawn_ik_rviz_control.py" output="screen">
  </node>

  <node pkg="relaxed_ik_ros1" type="relaxed_ik_rust.py" name="relaxed_ik_rust">
    <param name="simulation_time" value="go"/>
    <param name="use_visualization" value="True" />
  </node>

  <include file="$(find dawn_ik)/launch/collision_ik/collision_ik_adapter.launch" >
    <arg name="tf_prefix" value=""/>
    <arg name="robot_name" value="lite6"/>
    <arg name="base_frame" value="link_base"/>
    <arg name="eef_frame" value="link_eef"/>
    <arg name="controller_command_topic" value="/ufactory/lite6_traj_controller/command"/>
    <arg name="controller_state_topic" value="/ufactory/lite6_traj_controller/state"/>
  </include>

</launch>
